devito/data/allocators.py

```diff
@@ -128,6 +129,10 @@ class PosixAllocator(MemoryAllocator):
     @classmethod
     def initialize(cls):
         handle = find_library('c')
+
+        if handle is None and os.name == "posix" and sys.platform == "darwin":
```

Comment:
Can you add a comment along the lines of:

```
# Special case: on MacOS Big Sur ....
```

Reply:
Of course I can, sorry about that!